esterni/manuale-sviluppatore/components/tecnologie-interessate.tex

```diff
@@ -0,0 +1,59 @@
+\documentclass[../manuale-sviluppatore.tex]{subfiles}
+
+\begin{document}
+\subsection{Back End}%
+\label{sub:back-end}
+In questa sezione sono descritte le tecnologie scelte dopo una fase di analisi per lo sviluppo del back-end.
+
+\subsubsection{InfluxDB}%
+\label{sub:influxDB}
+Abbiamo deciso di utilizzare InfluxDB, un Time Series Database open source, viste le sue alte performance e le sue elevate capacità di scrittura e di sostenere carichi di interrogazioni 
+importanti. Usiamo questo database per salvare dati momentanei, non persistenti.
+% TODO estendere
+
+\subsubsection{MySql}%
+\label{sub:mysql}
+Abbiamo deciso di utilizzare MySql per il salvataggio di dati persistenti attraverso il modello relazionale, il quale garantisce affidabilità.
+% TODO estendere
+
+\subsubsection{Spring}%
+\label{sub:spring}
+Per sviluppare il back-end abbiamo deciso di utilizzare Spring, un framework concepito per la realizzazione di applicazioni enterprise, che soddisfa i requisiti
+di performance, sicurezza e affidabilità richiesti. \\
+Spring contiene molte librerie, che ci hanno consentito di risolvere in modo elegante più di qualche requisito.
+Unico lato negativo di questo framework è la sua documentazione, che può risultare molto dispersiva.
```

Comment:
> ci hanno consentito di risolvere in modo elegante più di qualche requisito.

_in modo elegante_, _più di qualche_, vanno bene in una conversazione tra di noi, ma a parere mio non è la forma più corretta per un manuale sviluppatore

Cambierei tutto in un più freddo e professionale:

Per sviluppare il back-end è stato utilizzato Spring, un framework concepito per la realizzazione di applicazioni enterprise, che soddisfa i requisiti di performance, sicurezza e affidabilità richiesti.
Spring è inoltre corredato da librerie che hanno permesso di soddisfare molti requisiti del progetto.